File: En Python - Interpretado/EliminarElementoDeUnaLista.py
#!/usr/bin/env python

def removeItems(lst, item):
    for i in lst:
        if i == item:
            lst.remove(item)
    return lst

lst = [2, 90, 89, 2, 5, 101, 90, 35, 2, 43, 89, 101, 5, 89, 90, 5]

print("Lista actual: {}".format(lst))

item = int(input("\nIngrese el elemento que desea eliminar: "))

print("Lista modificada: {}".format(removeItems(lst, item)))

File: En Kotlin - JVM/EliminarElementoDeUnaLista.kt
fun main(args: Array<String>) {
    val arrs = mutableListOf<Int>(2, 90, 89, 2, 5, 101, 90, 35, 2, 43, 89, 101, 5, 89, 90, 5)
    val item: Int

    println("Lista actual: $arrs")

    print("\n¿Que elemento desea eliminar?: ")
    item = readLine().toString().toInt()

    if(arrs.removeAll { it == item }) {
        println("Lista modificada: ${arrs}")
    } else {
        println("El elemento no existe!!")
    }
}
